coil-base/src/main/java/coil/util/Extensions.kt

```diff
@@ -195,3 +201,20 @@ internal inline operator fun MemoryCache.Key.Companion.invoke(
         parameters = parameters.cacheKeys()
     )
 }
+
+@OptIn(ExperimentalCoilApi::class)
+internal fun DefaultRequestOptions.copy(
```

Comment:
This is private to avoid having to maintain binary compatibility for it. It's also not likely to be used outside of the library.